operations/model.go

```diff
@@ -91,6 +91,23 @@ func NewClientSettings(fn string) (*ClientSettings, error) {
 	}
 	conf.LoadedFrom = path
 
+	// check for a local settings file
+	localPath, err := findConfigFilePath(".evergreen.yml")
+	if err != nil {
+		return conf, nil
+	}
+
+	localData, err := ioutil.ReadFile(localPath)
+	if err != nil {
+		return nil, errors.Wrap(err, "problem reading configuration from file")
```

Comment:
Error message should be clearer, something about expecting a local configuration in a particular filename.

Reply:
Done.